mcc/resources/schemas/dbscripts/postgresql/mcc-20.003-20.004.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE mcc.animalRequests (
```

Comment:
While placing database-level constraints isnt inherently a bad idea, I would suggest not doing this here. we can instead use <nullable>false</> in the schema XML. enforcing this in the query layer is easier in case we ever want to violate these rules. for example, maybe there will be a time when we want to allow people to save draft forms, or perhaps some of these fields will actually end up being nullable.

i'd just enforce DB-level not null for the PKs.